OsmAnd/src/net/osmand/plus/search/SearchUtils.java

```diff
@@ -51,8 +51,8 @@ public static PoiUIFilter getShowOnMapFilter(@NonNull OsmandApplication app, @No
 					}
 				} else {
 					filter = app.getPoiFilters().getSearchByNamePOIFilter();
```

Comment:
What SearchUtils is about? Feels wrong class design

Reply:
SearchUtils should be refactored no static methods combined / split with PoiUIFilter differently